VSRAD.DebugServer/IPC/StreamExtensions.cs

```diff
@@ -61,5 +63,139 @@ public static async Task<int> WriteSerializedMessageAsync<T>(this Stream stream,
                 return prefixedMessageLength;
             }
         }
+
+        public static async Task<bool> ProcessDataTransfer(this Stream writer, ICommand command)
+        {
+            switch (command)
+            {
+                case SendFileCommand _:
+                    var send_command = (SendFileCommand)command;
+                    var sendPath = Path.Combine(send_command.SrcPath, send_command.Metadata.relativePath_);
+                    return await SendFileAsync(writer, sendPath, send_command.UseCompression).ConfigureAwait(false);
+                case GetFileCommand _:
+                    var receive_command = (GetFileCommand)command;
+                    var receivePath = Path.Combine(receive_command.DstPath, receive_command.Metadata.relativePath_);
+                    return await ReceiveFileAsync(writer, receivePath, receive_command.UseCompression);
```

Comment:
In this chunk of code `switch` already does conversion for us, however we discard it using `_`, only to do it again manually in the next line. Also, we do not use snake case anywhere in project (as far as I know)

I suggest rewriting this `switch` like this:

```c#
case SendFileCommand sf:
    var sendPath = Path.Combine(sf.SrcPath, sf.Metadata.relativePath_);
    return await SendFileAsync(writer, sendPath, sf.UseCompression).ConfigureAwait(false);
case GetFileCommand gf:
    var receivePath = Path.Combine(gf.DstPath, gf.Metadata.relativePath_);
    return await ReceiveFileAsync(writer, receivePath, gf.UseCompression);
default:
    break;
```